### ISSUE TITLE
Fix apply require-jsdoc for ArrowFunctionExpression

### ISSUE THREAD

**Issue by winor30** [gajus/eslint-plugin-jsdoc#502]
It seemed that auto fixing by `require-jsdoc` is incorrect.
This bug add jsdoc comment to incorrect position in arrow function case.
If you would like to check, running the command in bellow with [the repository](https://github.com/winor30/eslint-jsdoc-arrow-function-test) is reproduction the bug.

```sh
$ cat index.js
const hello = name => {
  document.body.textContent = "Hello, " + name + "!";
};

hello("World");

$ npx eslint --fix index.js

$ cat index.js
const hello = /**
 *
 */
name => {
  document.body.textContent = "Hello, " + name + "!";
};

hello("World");
```

**Comment by brettz9** [comment]
Seems essentially the same as #403, so closing. Feel free to subscribe there, or if you think the issue is different (beyond being about arrow vs. non-arrow expressions), feel free to comment further here.

**Comment by brettz9** [comment]
Fixed by c1b5b46781f2356405c601c835c636e7cfc33ee7 (part of https://github.com/gajus/eslint-plugin-jsdoc/releases/tag/v24.0.1 )
